common/pkg/cgroups/systemd_linux.go

```diff
@@ -32,13 +32,7 @@ func systemdCreate(resources *cgroups.Resources, path string, c *systemdDbus.Con
 			systemdDbus.PropDescription("cgroup " + name),
 			systemdDbus.PropWants(slice),
 		}
-		var ioString string
-		v2, _ := IsCgroup2UnifiedMode()
-		if v2 {
-			ioString = "IOAccounting"
-		} else {
-			ioString = "BlockIOAccounting"
-		}
+		ioString := "IOAccounting"
```

Comment:
This can be inlined into the only user.

(Some similar opportunities exist elsewhere.)